src/base/bittorrent/cachestatus.cpp

```diff
@@ -50,7 +50,11 @@ qreal CacheStatus::readRatio() const
 
 int CacheStatus::jobQueueLength() const
 {
+#if LIBTORRENT_VERSION_NUM < 10100
     return m_nativeStatus.job_queue_length;
+#else
+    return m_nativeStatus.queued_jobs;
```

Comment:
This is only the first step, which simply allows to build the qBittorrent with this version of libtorrent but does not use its improvements/advantages. Next, I plan to go step by step on new features and abandon deprecated.